larray/core/array.py

```diff
@@ -3076,7 +3076,7 @@ def indexofmax(self, axis=None):
 
     posargmax = renamed_to(indexofmax, 'posargmax')
 
-    def labelsofsorted(self, axis=None, kind='quicksort'):
+    def labelsofsorted(self, axis=None, kind='quicksort', reverse=False):
```

Comment:
I wonder if reverse shouldn't be the second argument, as it is much more likely to be used than kind.